api/api/routers/mcp/_mcp_models.py

```diff
@@ -253,3 +253,46 @@ def from_model_data(cls, id: str, model: FinalModelData):
             )
 
     data: list[ModelItem]
+
+
+class Error(BaseModel):
+    code: str
+    message: str
+    details: dict[str, Any] | None  # not sure this is the correct format, but you get the idea
+
+
+class AgentVersion(BaseModel):
+    id: str
+    temperature: float
+    # ...
+
+
+class Run(BaseModel):
+    # none exhaustive list of fields, to get started.
+    id: str
+    conversation_id: str  # is there always a conversation_id?
+    agent_id: str
+    agent_schema_id: int
+    agent_version: AgentVersion
+    status: Literal[
+        "success",
+        "error",
+        "pending",
+        "running",
+        "cancelled",
+    ]  # not sure about the exact list of statuses, but you get the idea (we should use Pydantic every-where!)
+    agent_input: dict[str, Any] | None
+    # agent_output: dict[str, Any] (I don't think we should return a `agent_output`, the output is already part of the `llm_completions` field)
+    duration_seconds: float
+    cost_usd: float
+    created_at: datetime
+    metadata: dict[str, Any] | None  # very important
+    llm_completions: (
```

Comment:
list[Message]

Reply:
In case of multi turn conversations, do we want to include the "run_id" of intermediary assistant messages ? (similar to what is displayed in the frontend for chatbots for examples)